packages/design-system/src/styles.css

```diff
@@ -1,6 +1,117 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import 'tailwindcss' prefix(k);
```

Comment:
We wanna keep `k1` prefix to prevent potential conflicts.

Reply:
Nope:

```
Error: The prefix "k1" is invalid. Prefixes must be lowercase ASCII letters (a-z) only.
```

Reply:
`kone`? Too long? `ko`?